auth/standard_role_manager.cc

```diff
@@ -566,8 +566,8 @@ standard_role_manager::revoke(std::string_view revokee_name, std::string_view ro
             }
 
             return make_ready_future<>();
-        }).then([this, revokee_name, role_name] {
-            return this->legacy_modify_membership(revokee_name, role_name, membership_change::remove);
+        }).then([this, revokee_name, role_name, &mc] {
+            return this->modify_membership(revokee_name, role_name, membership_change::remove, mc);
```

Comment:
We can probably from `this->` while we are at it.

Reply:
ok